### PR TITLE
feat: bumped antd version, fixed depricated parts

### PR DESCRIPTION
**Pull request by georgesmithishere**
https://www.notion.so/eb96722b397d42c9908002ca06af88e3?v=1f519ba1849347aa9740faedda4acf3b&p=eae1b4be706349e28450ee06c09d161e

### CODE REVIEW

**Comment by r0x4xn** [comment]
App <strong>react-boilerplate</strong> has been deployed to Vercel.

<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://react-boilerplate-2867ne8vj-prepo.vercel.app'>https://react-boilerplate-2867ne8vj-prepo.vercel.app</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="prj_3Lydru5gdN5RuqWFQBT8AZrbiROq"></hidden>

**Comment by r0x4xn** [comment]
App <strong>core-dapp</strong> has been deployed to Vercel.

<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://core-dapp-83241svbf-prepo.vercel.app'>https://core-dapp-83241svbf-prepo.vercel.app</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="prj_JWVJ3WoV71UcOmB52qrcezjyBL3V"></hidden>

File: apps/frontend/core-dapp/src/components/Menu.tsx [hunk 1]
```diff
@@ -12,8 +13,30 @@ export const MenuItem: React.FC<MenuItemProps> = (props) => (
...
+      color: ${({ theme }): string => theme.color.neutral3};
+      font-weight: ${({ theme }): number => theme.fontWeight.semiBold};
+    }
+    :hover {
+      * {
+        color: ${({ theme }): string => theme.color.primary};
+        font-weight: ${({ theme }): number => theme.fontWeight.medium};
+      }
+    }
+`
```
Comment (chrisling-dev) [inline apps/frontend/core-dapp/src/components/Menu.tsx:32]: Missing one closing curly bracket here
```suggestion
const keepBackgroundOnHover = css`
  .ant-dropdown-menu-item-active {
    background-color: ${({ theme }): string => theme.color.neutral10};;
    * {
      color: ${({ theme }): string => theme.color.neutral3};
      font-weight: ${({ theme }): number => theme.fontWeight.semiBold};
    }
    :hover {
      * {
        color: ${({ theme }): string => theme.color.primary};
        font-weight: ${({ theme }): number => theme.fontWeight.medium};
      }
    }
    }
`
```

File: apps/frontend/core-dapp/src/components/Menu.tsx [hunk 2]
```diff
@@ -57,9 +80,12 @@ const Wrapper = styled.div<Props>`
...
     }
-    .ant-dropdown-menu-item-active {
-      background-color: ${({ theme }): string => theme.color.neutral7};
-    }
+    ${({ keepBackgroundColorOnHover }): FlattenInterpolation<ThemeProps<DefaultTheme>> => {
+      if (keepBackgroundColorOnHover) {
+        return keepBackgroundOnHover
+      }
+      return hoverBackground
+    }}
```
Comment (chrisling-dev) [inline apps/frontend/core-dapp/src/components/Menu.tsx:88]: Alternative shorter and cleaner code IMO

```suggestion
    ${({ keepBackgroundColorOnHover }): FlattenInterpolation<ThemeProps<DefaultTheme>> => keepBackgroundColorOnHover ? keepBackgroundOnHover : hoverBackground
    }
```

**Review by chrisling-dev — CHANGES_REQUESTED** [review]
Seems the color here is broken? 

Screenshot 2022-07-14 at 9 52 23 PM

**Review by chrisling-dev — CHANGES_REQUESTED** [review]

**Review by chrisling-dev — APPROVED** [review]

**Pull request merged**